change/@fluentui-react-badge-3bd9df4e-4e19-4ff6-b2bf-accfb6602308.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "fix: Match OOF icon mapping to v8.",
+  "packageName": "@fluentui/react-badge",
+  "email": "esteban.230@hotmail.com",
```

Comment:
shouldn't this be your microsoft e-mail?

Reply:
My account has both the microsoft one and my personal. Don't know how to change it, but if it's needed I can find it out.